dockstore-common/src/main/java/io/dockstore/common/Constants.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2015 Collaboratory
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.dockstore.common;
+
+/**
+ * 
+ * This describes all keys available in Dockstore config files.
+ *
+ * @author xliu
+ */
+public class Constants {
+    public static final String WEBSERVICE_BASE_PATH = "webservice.base_path";
+    public static final String WEBSERVICE_TOKEN = "webservice.token";
+    public static final String WEBSERVICE_TOKEN2 = "webservice.token2";
+    public static final String WEBSERVICE_EXTRA_FILES = "webservice.extra_files";
+
+    public static final String POSTGRES_HOST = "database.postgresHost";
+    public static final String POSTGRES_USERNAME = "database.postgresUser";
+    public static final String POSTGRES_PASSWORD = "database.postgresPass";
+    public static final String POSTGRES_DBNAME = "database.postgresDBName";
+    public static final String POSTGRES_MAX_CONNECTIONS = "database.maxConnections";
+
+    public static final String RABBIT_HOST = "rabbit.rabbitMQHost";
+    public static final String RABBIT_USERNAME = "rabbit.rabbitMQUser";
+    public static final String RABBIT_PASSWORD = "rabbit.rabbitMQPass";
+    public static final String RABBIT_QUEUE_NAME = "rabbit.rabbitMQQueueName";
+
+    public static final String PROVISION_MAX_RUNNING_CONTAINERS = "provision.max_running_containers";
```

Comment:
These variables seem copied from consonance, please delete all the ones that are not relevant to dockstore